dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -32,13 +32,46 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
 
+/**
+ * Option for {@link StaticLinkageChecker}. To construct an input class path, the checker requires
+ * either one of the following types of input:
+ *
+ * <ul>
+ *   <li>{@code bomCoordinate}: a Maven coordinate for a BOM
+ *   <li>{@code mavenCoordinates}: list of the coordinates of Maven artifacts
+ *   <li>{@code jarFileList}: list of jar files in the filesystem
+ * </ul>
+ *
+ * @see <a
+ *     href="https://github.com/GoogleCloudPlatform/cloud-opensource-java/tree/master/dependencies#input">Static
+ *     Linkage Checker: Input</a>
+ */
 @AutoValue
 abstract class StaticLinkageCheckOption {
   // TODO(suztomo): Add option to specify entry point classes
 
-  @Nullable abstract String getBomCoordinate();
+  /**
+   * Returns a Maven coordinate for a BOM if specified; otherwise null. Example value: {@code
+   * com.google.cloud:cloud-oss-bom:pom:1.0.0-SNAPSHOT}
+   */
+  @Nullable
+  abstract String getBomCoordinate();
```

Comment:
What happens if more than one of these is set?

Reply:
This needs to be specified in the code itself. Really, though, this needs to throw an exception if someone tries to set more than one. Either that or refactor somehow so that not more than one can be set.

Reply:
If you really want to make that happen, you could use `@AutoOneOf` (it's part of the AutoValue package).

But setting more than one flag is a flag validation problem, not programmer error. I think `parseArguments()` below should fail and report an error to the user if they specify an invalid combination of options.

Reply:
Now `parseArguments` throws `IllegalArgumentException` if the combination is invalid.